public/demo.html

```diff
@@ -1,19 +1,34 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width,initial-scale=1.0" />
     <title>humm-widgets-cards</title>
   </head>
   <body>
     <!-- This demo file is used to test both the production bundles after running yarn build -->
     <h1>Humm widgets cards online</h1>
 
-    <h2>AU widget</h2>
+    <h2>AU default widget</h2>
     <script src="./humm-widgets-cards-au.umd.js?productPrice=400"></script>
 
-    <h2>NZ widget</h2>
+    <h2>AU blank widget (remove other widgets to see this in action)</h2>
```

Comment:
blank widgets wont work unless other themed widgets are removed from the page. I've hence added a comment `(remove other widgets to see this in action)`